### PR TITLE
@dzucconi - artwork is not contactable if partner is limited fair partner

### PR DESCRIPTION
**Pull request by broskoski**

### CODE REVIEW

File: schema/artwork/index.js [hunk 1]
```diff
@@ -112,6 +112,7 @@ const ArtworkType = new GraphQLObjectType({
                 artwork.forsale &&
                 !_.isEmpty(artwork.partner) &&
                 !artwork.acquireable &&
+                !artwork.partner.has_limited_fair_partnership
```
Comment (dzucconi) [inline schema/artwork/index.js]: &&

**Comment by mzikherman** [comment]
Should we be using `inquireable` here? That's the goal of that property, to separate a work's availability (a simple property, controlled and set by the partner), with whether or not its 'inquireable' (ie- do we allow/encourage/forbid a conversation to happen about this work), which is where some messy business logic might be contained.

Maybe we can try to use it?

**Comment by dzucconi** [comment]
@mzikherman I'm absolutely happy to use that; is there any difference between these definitions?

**Comment by mzikherman** [comment]
This is the definition Gravity has: https://github.com/artsy/gravity/blob/master/app/models/domain/artwork.rb#L490-L493

This does result in artworks that are not for sale, but that are still inquireable, such as https://www.artsy.net/artwork/harmony-korine-skinamax-chex. However, I notice on Force we don't actually display an inquiry form (or 'Contact Specialist').

This seems to be ad odds with our 'Shared View Logic' document- https://docs.google.com/document/u/1/d/1kQSHhCiFWxfVkSeql3GQA7UbBbhpNe-UyQ-c6q95Uq0/edit

Is that a bug? Based on the 'single source of truth to crazy business logic' (ie- Gravity), it seems like a bug- this artwork is actually inquireable! But we don't show the inquiry form? If it's turned out that the logic has diverged (for instance, I guess there is no more 'Contact Specialist?'), and that doc is out of date, we should strive to get things consistent, and maintain the correctness and validity of our 'single source of truth' (which out of necessity is Gravity).

**Comment by mzikherman** [comment]
@dzucconi @broskoski some questions. I think it's really important to try to use `inquireable` and get things consistent again. Things are very inconsistent right now.

Q1- Looking at https://github.com/artsy/gravity/blob/master/app/models/domain/artwork.rb#L490-L493, this means any artworks at an auction ARE inquireable. However, `is_contactable` has the exact opposite logic. It's possible this 'band-aid' was put in due to some UI weirdness (double inquiry form?). What's actually correct, should we remove that check in Gravity?

Q2 - It requires a partner (that restriction is not in Gravity). What about 'Ask a Specialist?'. That could be a use case where a work is inquireable (maybe it's been consigned?) and we still want to encourage conversation.

Q3 - It is only true if the work is NOT acquireable (ie- ecommerce). This doesn't make a ton of business sense (IMO), b/c while a work can be purchased immediately, we might still want to encourage a conversation/inquiry about it. Did a business rule change where e-commerce works should not have an inquiry/contact action?

Q4 - Lots of works are not for sale, but still inquireable (described in my previous comment). That's a big use case of this flag which seems to not be respected by clients anymore? Again, not sure if a business rule changed but I was under the impression that we'd still want to encourage a conversation on these works, currently we can't (on web).


I know this is complicated issue where it may be tempting to just go with 'what works', and not propagate changes upstream, but that leads to inconsistencies between how clients choose to handle this logic, and it'll be hard to ever consolidate. Additionally, it means adding 'messy' business logic around inquireable status (like the new limited fair partner) is complicated, breaks other clients, requires updates in many places, etc.

I feel strongly about trying to consolidate this, Metaphysics is the perfect place to get things on the same page again. Let me know what I can do to help this.

**Pull request merged**